### PR TITLE
SRCH-4572 SRCH-4574 SRCH-4575 Add total counts

### PR DESCRIPTION
**Pull request by stevenbarragan**
## Summary
- Send results total to the react and displayed it on results page

### CODE REVIEW

File: app/javascript/components/Results/ResultsCount/ResultsCount.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable camelcase */
```
Comment (ddzz) [inline app/javascript/components/Results/ResultsCount/ResultsCount.tsx:1]: Why is this `eslint-disable` needed?

File: app/javascript/components/SearchResultsLayout.tsx [hunk 2]
```diff
@@ -199,6 +200,7 @@ const SearchResultsLayout = ({ page, resultsData, additionalResults, vertical, p
             results={resultsData.results}
             vertical={vertical}
             totalPages={resultsData.totalPages}
+            total={resultsData?.total}
```
Comment (ddzz) [inline app/javascript/components/SearchResultsLayout.tsx:203]: Can this be:
```suggestion
            total={resultsData.total}
```

File: app/javascript/utils/index.ts [hunk 3]
```diff
@@ -56,3 +56,7 @@ export const move = <T>(input: T[], from: number, to: number): T[] => {
 
   return input.splice(to, numberOfDeletedElm, elm);
 };
+
+export const numberWithDelimiter = (number: number): string => {
+  return number.toLocaleString('en-US');
```
Comment (ddzz) [inline app/javascript/utils/index.ts:61]: Are we fine with a hard-coded `en-US` locale for this?

**Comment by ddzz** [comment]
Looks good, just had some minor questions.

File: app/javascript/components/Results/ResultsCount/ResultsCount.tsx (hunk 1 above)
Comment (stevenbarragan) [reply app/javascript/components/Results/ResultsCount/ResultsCount.tsx:1]: It's needed to used the translation in here https://github.com/GSA/search-gov/pull/1368/files/da43b3e7dbb1a57314a5fdd9e0318665e52ddd6c#diff-14cde637abda389b07de93f02d2b612bc8a89c24b753322cc60b825693a01646R21

**Review by ddzz — DISMISSED** [review]
LGTM

**Review by ddzz — APPROVED** [review]

**Pull request merged**